demos/dashboards-find-similar-screener/demo.js

```diff
@@ -0,0 +1,112 @@
+const loadingLabel = document.getElementById('loading-label');
+
+function displayFindSimilarScreener (postmanJSON) {
+    const secIds = [
+        'SecId',
+        'Name',
+        'riskrating'
+    ];
+
+    const headerFormats = {
+        SecId: 'Security ID',
+        Name: 'Investment Name',
+        riskrating: 'Risk Rating'
+    };
+
+    const columns = secIds.map(id => ({
+        id: `InvestmentScreener_${id}`,
+        header: {
+            format: headerFormats[id] || id
+        }
+    }));
+
+    const board = Dashboards.board('container', {
+        dataPool: {
+            connectors: [
+                {
+                    id: 'find-similar-screener',
+                    type: 'MorningstarInvestmentScreener',
+                    options: {
```

Comment:
It seems some options are missing from original [docs](https://developer.morningstar.com/direct-web-services/documentation/direct-web-services/screener/find-similar): `&sortOrder=Name+Asc&CategoryId:EQ:EUCA000591%7COngoingCharge:LT:0&`

PS: Feel free to replace `EUCA000591` with something else (eg SPY) + add description to the demo that the returned list is similar to the chosen fund